fedimint-client/src/lib.rs

```diff
@@ -232,9 +231,9 @@ impl DynGlobalClientContext {
         update_stream
             .filter_map(|tx_update| {
                 std::future::ready(match tx_update.state {
-                    TxSubmissionStates::Accepted { txid } if txid == query_txid => Some(Ok(())),
-                    TxSubmissionStates::Rejected { txid, error } if txid == query_txid => {
-                        Some(Err(error))
```

Comment:
Please don't just throw away error messages, these are vital for debugging.

Reply:
I had to throw it away since we do not return it via the api endpoint anymore, @elsirion already commented on this

Reply:
We talked about this on a call and it makes sense, what do you think is the best way to get some insight into failures back though? Not having any client-side debug info will be a pain once there are independent federations where we can't see the logs. A custom query strategy that is a mix of union (for errors) and threshold (for acceptance) could work, right?